client/my-sites/current-site/notice.jsx

```diff
@@ -96,7 +96,7 @@ export class SiteNotice extends React.Component {
 	}
 
 	freeToPaidPlanNotice() {
-		if ( ! this.props.isEligibleForFreeToPaidUpsell ) {
+		if ( ! this.props.isEligibleForFreeToPaidUpsell || this.props.isDomainOnly ) {
```

Comment:
This seems like something that should be fixed in the selector instead:
https://github.com/Automattic/wp-calypso/blob/59bdfeeb97eda4266ad39410cb0a074d2c88dbc8/client/state/selectors/is-eligible-for-free-to-paid-upsell.js#L20

Reply:
Disagree. From a perspective of how the function is used, yes. From a perspective of context, this site `isEligibleForFreeToPaidUpsell`, it's just that the wording/links don't work, and I opted out of fixing it as we already have upsell on it.

Reply:
Not feeling strongly about this - there are pros and cons to both approaches. We can go with this one 👍